tcl/board.tcl

```diff
@@ -1528,7 +1555,7 @@ proc ::board::drawText {w sq text color args {shadow ""} } {
 }
 
 # Highlight last move played by drawing a rectangle around the two squares and/or an arrow
-proc  ::board::lastMoveHighlight {w moveuci} {
+proc  ::board::lastMoveHighlight {w moveuci {nag ""}} {
   $w.bd delete highlightLastMove
   if { ! $::highlightLastMove && ! $::arrowLastMove } {return}
```

Comment:
This line should be removed (highlighlastMoveNag does not work if the other options are all inactive).